tsl/test/sql/dist_hypertable.sql.in

```diff
@@ -843,22 +843,22 @@ SELECT (_timescaledb_internal.show_chunk(show_chunks)).*
 FROM show_chunks('disttable_drop_chunks');
 $$);
 
--- test passing newer_than as interval and cascade
-SELECT * FROM drop_chunks(newer_than => interval '10 years', table_name => 'disttable_drop_chunks', cascade => true);
+-- test passing newer_than as interval
+SELECT * FROM drop_chunks(newer_than => interval '10 years', table_name => 'disttable_drop_chunks');
 SELECT * FROM disttable_drop_chunks;
 
-CREATE TABLE "weird nAme\\#^."(time bigint, device int CHECK (device > 0), color int, PRIMARY KEY (time,device));
-SELECT * FROM create_distributed_hypertable('"weird nAme\\#^."', 'time', 'device', 3, chunk_time_interval => 100, replication_factor => 2);
+CREATE TABLE "weird nAme\\#^"(time bigint, device int CHECK (device > 0), color int, PRIMARY KEY (time,device));
```

Comment:
Why is this period removed here? This seems like an unrelated and unexplained change for this PR.

Reply:
The period is removed because the name need to be a valid REGCLASS identifier, and periods cannot be part of that and will give a syntax error.

```
mats=# CREATE TABLE "foo$#^." ();
CREATE TABLE
mats=# SELECT 'foo$#^.'::regclass;
ERROR:  invalid name syntax
LINE 1: SELECT 'foo$#^.'::regclass;
               ^
mats=# CREATE TABLE "foo$#^" ();
CREATE TABLE
mats=# SELECT 'foo$#^'::regclass;
 regclass 
----------
 "foo$#^"
(1 row)
```

Reply:
It is necessary to use double quotes around the weird name, so it works. In the above example, it will be:
```sql
SELECT '"foo$#^."'::regclass;
```

Reply:
```sql
postgres=# create table "foo$#^." ();
CREATE TABLE
postgres=# select '"foo$#^."'::regclass;
 regclass
-----------
 "foo$#^."
(1 row)

```

Reply:
@mkindahl I am not following. In the existing test, the table is successfully created using the name with the period in it (no syntax error), so it seems it is a valid name. If your issue is casting to a regclass, then I guess what @k-rus suggest is the proper way to do it, not changing the name.

Reply:
It just replaces one unrelated change with another. The reason we had this strange table name in the first place is because we parsed it ourselves and we wanted to check that it worked. Now we leave that the `regclassin` so the test is not really useful and the actual name is not relevant since it is "whatever reclassin handles". I just fixed the immediate problem in the fastest possible way since I did not consider this test nor what specific name used particularly important.

Anyway, I put the dot back in the name.